@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -1,121 +1,153 @@
 .navds-combobox__wrapper {
-  --__ac-combobox-icon-size: 1.5rem;
-  --__ac-combobox-wrapper-inner-padding: var(--a-spacing-2);
-  --__ac-combobox-list-item-padding-block: var(--a-spacing-3);
-  --__ac-combobox-list-item-padding-inline: var(--a-spacing-3);
-  --__ac-combobox-border-width: 1px;
-  --__ac-combobox-input-height: 2rem;
+  --__axc-combobox-icon-size: 1.5rem;
+  --__axc-combobox-wrapper-inner-padding: var(--ax-spacing-2);
+  --__axc-combobox-list-item-padding-block: var(--ax-spacing-3);
+  --__axc-combobox-list-item-padding-inline: var(--ax-spacing-3);
+  --__axc-combobox-border-width: 1px;
+  --__axc-combobox-input-height: 2rem;
 
   display: flex;
   flex-direction: column;
   width: 100%;
   position: relative;
+  border-radius: var(--ax-border-radius-medium);
+}
+
+.navds-form-field:not(:is(.navds-combobox--disabled, .navds-combobox--readonly)) {
+  & .navds-combobox__wrapper:hover {
+    border-color: var(--ax-border-focus);
+  }
 }
 
 .navds-form-field--small .navds-combobox__wrapper {
-  --__ac-combobox-icon-size: 1.25rem;
-  --__ac-combobox-wrapper-inner-padding: var(--a-spacing-1);
-  --__ac-combobox-list-item-padding-block: var(--a-spacing-1-alt);
-  --__ac-combobox-list-item-padding-inline: var(--a-spacing-2);
-  --__ac-combobox-input-height: 1.5rem;
+  --__axc-combobox-icon-size: 1.25rem;
+  --__axc-combobox-wrapper-inner-padding: var(--ax-spacing-1);
+  --__axc-combobox-list-item-padding-block: var(--ax-spacing-1-alt);
+  --__axc-combobox-list-item-padding-inline: var(--ax-spacing-2);
+  --__axc-combobox-input-height: 1.5rem;
 }
 
 .navds-combobox--disabled {
   opacity: 0.3;
-}
 
-.navds-combobox--disabled .navds-combobox__wrapper *:hover {
-  cursor: not-allowed;
-}
+  & .navds-combobox__wrapper {
+    & *:hover {
+      cursor: not-allowed;
+    }
+  }
 
-.navds-combobox--disabled .navds-text-field__input {
-  border: 1px solid var(--a-border-default);
-}
+  & .navds-combobox__selected-options {
+    pointer-events: none;
+  }
 
-.navds-combobox--disabled .navds-combobox__selected-options {
-  pointer-events: none;
+  & .navds-combobox--readonly {
+    pointer-events: none;
+  }
 }
 
 .navds-combobox--readonly {
   pointer-events: none;
-}
 
-.navds-combobox--readonly .navds-combobox__button-toggle-list {
-  color: var(--a-gray-500);
-}
+  & .navds-combobox__selected-options {
+    & .navds-chips__chip {
+      /* TODO: remove !important and add readonly prop to Chips as well? == code changes */
+      background-color: var(--ax-bg-neutral-moderateA) !important;
```

Comment:
@KenAJoh I'm not a fan of important at all... so I'm thinking the solution here is to have a `readonly` prop on Chips as well? Thoughts? This CSS should live there / closest to its own component I think.

Either that or some other CSS here that sets higher specificity without having to use `!important` 🥲

Reply:
Currently I think this is the only instance of a component rendering _another_ component inside itself with a readonly state?

Reply:
No, currently, removing the `!important` is too low specificity. also tried to add a `:root` at the top selector 🤔. (your suggestion is not supposed to have a `:hover` is it?)

Reply:
ah, I think it's due to layering 
![image](https://github.com/user-attachments/assets/b4b42800-ddfa-41fe-90d2-5c6265f0e621)

aksel.form.components is after aksel.components? 🤔 (maybe?)

Reply:
yeah, if i do this
![image](https://github.com/user-attachments/assets/055e66d1-0438-42c3-8194-a7fe1e568c92)
it works as intended... but this is probably not a great idea?... or is it? (need to check every other component / form component for breaking styling at least?)

Reply:
I wish GH comments would AJAX harder faster & stronger, i'll answer something that you already answered a whole minute after 😢 ... refreshing the page loses long scroll position :old_man_yells_at_github:

Reply:
Then this problem will get fixed with that PR! resolving this ✅ 🎉